app/models/review.rb

```diff
@@ -1,7 +1,17 @@
 class Review < ActiveRecord::Base
   belongs_to :restaurant
   belongs_to :user
+  has_many :votes
+
   validates :body, presence: true
   validates :rating, presence: true
   RATINGS = [1, 2, 3, 4, 5]
+
+  def score
+    review_score = 0
+    votes.each do |vote|
```

Comment:
alternatively, consider using a `counter_cache` so you don't have to query this directly every time. read upon counter caches @ http://guides.rubyonrails.org/association_basics.html